### PR TITLE
Bumped miniflare version to avoid CVE-2023-7078

### PR DESCRIPTION
**Pull request by maddsua**
## Changes

Bumped miniflare version (3.20231025.1 --> 3.20231030.2)
https://github.com/advisories/GHSA-fwvg-2739-22v7

## Testing

No changes to the package itself and the fixes to miniflare should not affect this adapter.

## Docs

No docs update is needed as this PR only fixes that annoying npm vuln warning

### CODE REVIEW

**Review by alexanderniebuhr — APPROVED** [review]
@maddsua Thank you, for the contribution! ❤️

File: packages/cloudflare/package.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "//comment": "test changeset-bot",
   "name": "@astrojs/cloudflare",
   "description": "Deploy your site to Cloudflare Workers/Pages",
-  "version": "8.0.1",
+  "version": "8.0.2",
```
Comment (alexanderniebuhr) [inline packages/cloudflare/package.json:5]:
```suggestion
  "version": "8.0.1",
```

**Comment by maddsua** [comment]
Yeah, right, I shouldn't have bumped package version manually. Thanks for helping to do it correctly @alexanderniebuhr !

**Comment by alexanderniebuhr** [comment]
I'm sorry that I messed up the merge...
Will do a release soon :)

**Pull request merged**